### PR TITLE
Added escape hatch feature

### PR DESCRIPTION
**Pull request by Dev-Mikko**
Added the escape hatch feature to the edumeet-client. A button is placed inside the ListMe component. If all the joined peers in the meeting click on this button, the meeting will be ended immediately.

Here a screenshot of the feature:

immagine

### CODE REVIEW

File: src/store/middlewares/peerMiddleware.tsx [hunk 1]
```diff
@@ -35,6 +35,8 @@ const createPeerMiddleware = ({
 									roles,
 									raisedHand,
 									raisedHandTimestamp,
+									escapeMeeting,
```
Comment (havfo) [inline src/store/middlewares/peerMiddleware.tsx:38]: Remove all knowledge about the escape state of other peers, only the server has to know.

**Review by havfo — CHANGES_REQUESTED** [review]
Looks good apart from the comment :+1:

File: src/store/middlewares/peerMiddleware.tsx (hunk 1 above)
Comment (Dev-Mikko) [reply src/store/middlewares/peerMiddleware.tsx]: Done

**Review by havfo — APPROVED** [review]
LGTM

**Pull request merged**